### PR TITLE
Problem: cometbft rpc is more efficient to broadcast tx

### PR DESCRIPTION
**Pull request by yihuang**
👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

## Summary by CodeRabbit

- **New Features**
  - Added functionality for generating Cosmos transactions.
  - Introduced a new benchmark for Cosmos broadcast RPC.
  
- **Bug Fixes**
  - Fixed issues with running a single validator benchmark locally.
  - Resolved a problem preventing node shutdown via signal.
  
- **Improvements**
  - Enhanced transaction handling with parallel test transaction generation.
  - Updated load generator to retry with backoff on errors.
  - Added abort operation in the PrepareProposal process.
  
- **Documentation**
  - Updated CHANGELOG.md with new entries for bug fixes and improvements.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@
 * [#1645](https://github.com/crypto-org-chain/cronos/pull/1645) Gen test tx in parallel even in single node.
 * (testground)[#1644](https://github.com/crypto-org-chain/cronos/pull/1644) load generator retry with backoff on error.
 * [#1648](https://github.com/crypto-org-chain/cronos/pull/1648) Add abort OE in PrepareProposal.
+* (testground)[#]() Benchmark use cosmos broadcast rpc.
```
Comment (yihuang) [inline CHANGELOG.md:15]:
```suggestion
* (testground)[#1651](https://github.com/crypto-org-chain/cronos/pull/1651) Benchmark use cosmos broadcast rpc.
```

File: testground/benchmark/benchmark/cosmostx.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    signatures = Field("bytes", 3, repeated=True)
+
+
+class MsgEthereumTx(ProtoEntity):
+    from_ = Field("bytes", 5)
+    raw = Field("bytes", 6)
+
+
+class ExtensionOptionsEthereumTx(ProtoEntity):
+    pass
```
Comment (yihuang) [inline testground/benchmark/benchmark/cosmostx.py:86]:
```suggestion
```

File: testground/benchmark/benchmark/cosmostx.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    if msg is not None:
+        value = msg.SerializeToString()
+    return ProtoAny(type_url=type_url, value=value)
+
+
+class TxBody(ProtoEntity):
+    messages = Field(ProtoAny, 1, repeated=True)
+    memo = Field("string", 2)
+    timeout_height = Field("uint64", 3)
+    extension_options = Field(ProtoAny, 1023, repeated=True)
```
Comment (mmsqe) [inline testground/benchmark/benchmark/cosmostx.py:22]: isn't 4 here?

Comment (yihuang) [reply testground/benchmark/benchmark/cosmostx.py:22]: https://github.com/cosmos/cosmos-sdk/blob/main/proto/cosmos/tx/v1beta1/tx.proto#L138

**Pull request merged**